src/style.css

```diff
@@ -1 +1,23 @@
 /* add styles here */
+
+@import './blocks/big-search.css';
+@import './blocks/small-search.css';
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Avenir, system-ui;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf), url(./fonts/Avenir-Heavy.ttf),
+    url(./fonts/Avenir.ttc);
```

Comment:
The `@font-face` rule should specify the format for each font URL to ensure proper loading across different browsers. Consider adding format descriptors like `format('truetype')` or `format('opentype')`.